docs/geedocs/docsrc/answer/troubleshooting/fusionIssues.rst

```diff
@@ -6,11 +6,11 @@ Fusion issues
 
 .. container::
 
-      .. container:: content
+   .. container:: content
 
       When you install GEE on a server, the software does a query for
       ``hostname -f``. Each time you build a resource, project, or
-      database on the Fusion server, the initial ``hostname -f``
+      dgeecheckTestServerStatus.rstatabase on the Fusion server, the initial ``hostname -f``

```

Comment:
looks like this should be database

Reply:
yes, some cut and paste error. Good catch. I will fix it and push a commit